rules/meaningful-pbi-titles/rule.md

```diff
@@ -14,9 +14,9 @@ archivedreason: null
 guid: 47cff095-9ee2-432f-bac4-4cbbeb2314b5
 ---
 
-Product Backlog Items (PBIs) are the cornerstone of a well-oiled project. They track features, bugs, tasks, and much more. When a developer or Product Owner is looking through the backlog, it's important that - at a glance -  they can read the titles of PBIs and have a decent understanding of them.
+Product Backlog Items (PBIs) and Issues are the cornerstone of a well-oiled project. They track features, bugs, tasks, and much more. When a developer or Product Owner is looking through the backlog, it's important that - at a glance -  they can read the titles of PBIs and have a decent understanding of them.
 
-So what separates a **good** PBI title from a **bad** one?
+So what separates a **good** PBI/Issue title from a **bad** one?
```

Comment:
PBI is a Scrum term, not specific to DevOps. GitHub issues are a form of a PBI.
Typically I'll use the term PBI to cover any form of PBI (DevOps, Trello, GitHub - Any backlog item)

1. Action if you agree